app/Http/Controllers/ApiPhotosController.php

```diff
@@ -148,6 +151,12 @@ protected function storePhoto(Request $request): Photo
 
         $date = Carbon::parse($date);
 
+        if (app()->environment() === "production") {
+            if (Photo::where(['user_id' => $user->id, 'datetime' => $date])->exists()) {
+                throw new Exception("You have already uploaded this file!");
```

Comment:
should we return something more generic to the mobile app, like ['success' => false, msg => 'exists']` ?

Reply:
Also, please allow user_id = 1 to duplicate images on production, sometimes I need this for testing

Reply:
The mobile app does not need the 'success' => false info, as the request status will be 500 or 422. The message is already there. What do we get with this current implementation on mobile?

I'm allowing the user_id=1 on the next bugfix PR.

Reply:
I checked on mobile, nothing shows to the user, just that Something went wrong. Need to discuss this with Anoop to see what he needs on his side.